testing/test-env.tmpl.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 Google LLC
```

Comment:
Is this file complete?

Reply:
I've only included the env vars I've been using, not sure if that's typical or not.